unittests/scans/dependency_track/many_findings_with_alias.json

```diff
@@ -585,21 +585,11 @@
             "severity": "HIGH",
             "severityRank": 1,
             "cweName": "Uncontrolled Resource Consumption",
-            "vulnId": "GHSA-95rf-557x-44g5",
-            "aliases": [
-               {
-                  "allBySource": {
-                     "NVD": "CVE-2022-2053",
-                     "GITHUB": "GHSA-95rf-557x-44g5"
-                  },
-                  "cveId": "CVE-2022-2053",
-                  "ghsaId": "GHSA-95rf-557x-44g5",
-                  "id": 6883
-               }
-            ],
+            "vulnId": "CVE-2022-2053",
+            "aliases": [],
```

Comment:
You should not modify existing unit tests but create a new one for this bug.

Reply:
fair enough, created a new test for this